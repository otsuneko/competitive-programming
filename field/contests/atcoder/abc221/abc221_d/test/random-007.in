10
8 5
4 10
5 3
8 8
2 9
10 6
3 7
6 5
1 2
2 7
