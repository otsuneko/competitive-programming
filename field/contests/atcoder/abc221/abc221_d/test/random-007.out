1 2 3 3 1 4 1 0 0 0
